cdap-api/src/main/java/co/cask/cdap/api/annotation/Batch.java

```diff
@@ -99,9 +99,12 @@
  * </p>
  *
  * @see HashPartition
+ *
+ * @deprecated As of release 5.0.0, implement {@link @ProgramType.SPARK} for realtime processing.
```

Comment:
Instead of implement, maybe say "replaced by"?

Reply:
Addressed comments